apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -96,6 +102,19 @@ defmodule Aecore.Miner.Worker do
     mine_sync_block(Pow.generate(cheader), cblock)
   end
 
+  defp mine_sync_block(
```

Comment:
Is this case needed when we have the check for != {:error, :miner_was_stopped} above?